src/views/AgmLocationChg.vue

```diff
@@ -14,15 +14,6 @@
       @exit="onPaymentErrorDialogExit()"
     />
 
-    <StaffPaymentDialog
-      :staffPaymentData.sync="staffPaymentData"
-      attach="#agm-location-chg"
-      :dialog="staffPaymentDialog"
-      :loading="filingPaying"
-      @exit="staffPaymentDialog=false"
-      @submit="onClickFilePay(true)"
-    />
```

Comment:
Can staff still file an AGM Location Change?

Reply:
Yes Sev! I just tested that too.